scripts/pathUtil.js

```diff
@@ -0,0 +1,31 @@
+const fs = require("fs");
+const path = require("path");
+
+module.exports = function(useBdRelease, releaseInput, release, args) {
```

Comment:
the export should be named
```suggestion
module.exports = {
  getDiscordPath
};

function getDiscordPath(useBdRelease, releaseInput, release, args) {
```

Reply:
Fixed (Amended commit)